dist/s_style

```diff
@@ -23,6 +23,11 @@ if [ $# -ne 1 ]; then
 	    -e '/checksum\/power8/d' \
 	    -e '/checksum\/zseries/d' |
 	xargs $xp -n 1 -I{} sh ./dist/s_style {}
+
+	# Checks source files for camel case formatting, variables should be separated by underscore instead.
+	if egrep -n -r '\b[a-z]+[A-Z]' src | egrep -v '/os_win/|/docs/|/tags|:[     ]+\*|"|hash_city|huffman|checksum|UNCHECKED_STRING'; then
```

Comment:
@mm-ng , I would suggest we follow the existing way of running the tests per file. So basically moving the code in the else portion and checking for whether to exclude the files in there.